.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'm not too keen on adding this file to source control. In my view, this is veering into personal preference territory and I have no interest in trying to find a common ground amongst everyone contributing. My personal `launch.json` looks quite a bit different which I suspect will not suite anyone besides myself.

Reply:
I agree, the only reason I left it in was that the default run task tries to run whatever file you're looking at, which always fails unless you go back to main.go.

I'd be curious to see your launch.json :)

Reply:
Nothing too crazy, just very specific.

I have two primary configs to choose between the current file or the current workspace. I find the file config works well with for running tests within a specific file and the workspace config when I need to launch the app - which is honestly rare for me.

I then have many profiles with environment configs to quickly launch with a specific config profile. I find that much easier than swapping out the yaml file.
 
```
{
    "version": "0.2.0",
    "configurations": [{
        "name": "File",
        "type": "go",
        "request": "launch",
        "mode": "auto",
        "program": "${fileDirname}",
        "env": {},
        "args": []
    },
    {
        "name": "Workspace",
        "type": "go",
        "request": "launch",
        "mode": "auto",
        "program": "${workspaceFolder}",
        "env": {},
        "args": []
    }]
}
```